src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -644,6 +644,63 @@ public void TestGenerateEvent_From_AuthFlowResult_With_TokenResult_And_Null_Erro
             eventData.Measures.Should().ContainKey("duration_milliseconds");
         }
 
+        /// <summary>
+        /// Test to validate user interactive env.
+        /// </summary>

```

Comment:
These XML Summaries should not be required by StyleCop - so unless they describe extra context the test name can't give, I would prefer leaving them off.

Reply:
I was getting an error so included them. Suppressed the rule for test projects.